chore: minor readme tweaks (#315)
cosmos/solidity-ibc-eureka: README.md

> # IBC v2 in Solidity  [![Github Actions][gha-badge]][gha] [![Foundry][foundry-badge]][foundry] [![License: MIT][license-badge]][license] [![Code Coverage][codecov-badge]][codecov]

Reviewer: ```suggestion
# IBC in Solidity  [![Github Actions][gha-badge]][gha] [![Foundry][foundry-badge]][foundry] [![License: MIT][license-badge]][license] [![Code Coverage][codecov-badge]][codecov]
```

| **Contracts** | **Description** | **Status** |
|:---:|:---:|:---:|

> | `ICS26Router.sol` | IBC v2 router handles sequencing, replay protection, and timeout checks. Passes proofs to light clients for verification, and resolves `portId` for app callbacks. Provable IBC storage is stored in this contract.  | ✅ |
> | `ICS20Transfer.sol` | IBC v2 transfer application to send and receive tokens to/from another IBC v2 transfer implementation. | ✅ |

Reviewer: ```suggestion
| `ICS26Router.sol` | IBC router handles sequencing, replay protection, and timeout checks. Passes proofs to light clients for verification, and resolves `portId` for app callbacks. Provable IBC storage is stored in this contract.  | ✅ |
| `ICS20Transfer.sol` | IBC transfer application to send and receive tokens to/from another IBC v2 transfer implementation. | ✅ |
```

| `SP1ICS07Tendermint.sol` | The light client contract, and the entry point for SP1 proofs. | ✅ |

> | `ICS27Controller.sol` | IBC v2 interchain accounts controller. | ❌ |
> | `ICS27Host.sol` | IBC v2 interchain accounts host. | ❌ |

Reviewer: ```suggestion
| `ICS27Controller.sol` | IBC interchain accounts controller. | ❌ |
| `ICS27Host.sol` | IBC interchain accounts host. | ❌ |
```

There are three test suites in the `e2e/interchaintestv8` directory:

> - `TestWithIbcEurekaTestSuite`: This test suite tests the IBC v2 contracts via manual relaying (requires the operator to be installed).

Reviewer: ```suggestion
- `TestWithIbcEurekaTestSuite`: This test suite tests the IBC v2 contracts via the relayer (requires the operator and the relayer to be installed).
```

    - To run any of the tests, run the following command:
        ```sh
        just test-e2e $TEST_NAME
        ```

> - `TestWithRelayerTestSuite`: This test suite tests the IBC v2 contracts via the relayer (requires the relayer and operator to be installed).

Reviewer: ```suggestion
- `TestWithRelayerTestSuite`: This test suite tests the relayer via the IBC contracts (requires the relayer and operator to be installed).
```

> - `TestWithRelayerTestSuite`: This test suite tests the IBC v2 contracts via the relayer (requires the relayer and operator to be installed).

Reviewer: We should add the Comsos to cosmos test suite in this list as well.
